graylog2-web-interface/src/views/components/TimerangeForForm.ts

```diff
@@ -22,8 +22,21 @@ import type { TimeRange } from 'views/logic/queries/Query';
 import type { SearchBarFormValues } from 'views/Constants';
 import { isTypeRelativeWithStartOnly, isTypeRelativeWithEnd } from 'views/typeGuards/timeRange';
 
+import { TimeRangeDropDownFormValues } from './searchbar/date-time-picker/TimeRangeDropdown';
+
 const formatDatetime = (datetime) => datetime.toString(DateTime.Formats.TIMESTAMP);
 
+export const onSettingSearchBarFormTimeRange = (nextTimeRange: TimeRangeDropDownFormValues['nextTimeRange']) => {
+  if (isTypeRelativeWithEnd(nextTimeRange) && nextTimeRange.from === RELATIVE_ALL_TIME) {
```

Comment:
I think we can also create a `isAllMessagesRange` predicate. This way we can also encapsulate the usage of the constant.

Reply:
Good idea, I created a helper function, but did not started to implement it everywhere, to keep this PR small.
I am not completely sure about the best place for this type of functions and created a separate file in `views/logic/queries`.